### ISSUE TITLE
Feat: Short the text in events cards and add a new page to display full details

### ISSUE THREAD

**Issue by Rahulm2310** [anitab-org/anitab-org.github.io#237]
### Is your feature request related to a problem? Please describe.
Currently, full text is displayed in the event cards.
![anitab-events-card-issue](https://user-images.githubusercontent.com/54268438/108544569-a9c51780-730c-11eb-85e6-1431551497f0.gif)

### Describe the solution you'd like
We should shorten the text in the events cards, add a `Read more` button in the card and add a new event details page to display full details of event.
Something like this can be achieved.
![image](https://user-images.githubusercontent.com/54268438/108544940-2a841380-730d-11eb-9f92-bdeb9a9e5560.png)

**Comment by Rahulm2310** [comment]
@annabauza @nandini45 What do you guys think?

**Comment by nandini45** [comment]
agreed @Rahulm2310  and also we may need to shorten the content its way too long

**Comment by Rahulm2310** [comment]
@annabauza @nandini45 If approved, I would like take this up.

**Comment by nandini45** [comment]
@Rahulm2310  i think i misread the solution firstly. 
please do as needed 
thanks for the work

**Comment by Rahulm2310** [comment]
@nandini45 I have tried to tackle this. Here are my changes. Please share your thoughts on this.
![anitab-events-fix](https://user-images.githubusercontent.com/54268438/109344664-0a1c0200-7895-11eb-864a-b8572efc9cdb.gif)

**Comment by nandini45** [comment]
great work 
just one question have you changed the corner edges 
if yes i think not all the corner have being changed, the size of the corner edges should be same for all edges
![image](https://user-images.githubusercontent.com/62046625/109398517-736a4680-7963-11eb-83a3-e25d5b9d961c.png)

**Comment by Rahulm2310** [comment]
@nandini45 I have not changed the edges. It is also there in the live project as well. 
![image](https://user-images.githubusercontent.com/54268438/109398610-00ad9b00-7964-11eb-876d-d01a3d6c26e8.png)
This is a screenshot from the live project.

**Comment by nandini45** [comment]
ok @Rahulm2310 
you want to create a issue of it
if yes go ahead. it can be quite an easy 1st timer issue if you want
and for your work
its perfect please create a PR

**Comment by Rahulm2310** [comment]
Sure. Thanks @nandini45 .
